src/test/java/nextstep/courses/domain/CourseTest.java

```diff
@@ -0,0 +1,67 @@
+package nextstep.courses.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import nextstep.users.domain.NsUserTest;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.internal.bytebuddy.asm.Advice.Local;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CourseTest {
+
+    Image image;
+
+    @BeforeEach
+    void setUp() {
+        image = new Image("test.png", 300, 200, 1_000);
+    }
+
+    @Test
+    void 무료강의_생성() {
+        Course course = new Course();
+        LocalDateTime now = LocalDateTime.now();
+
+        Session session1 = new Session.Builder(1L)
+                .title("lms")
+                .sessionType(SessionType.FREE)
+                .image(image)
+                .state(SessionState.RECRUITING)
+                .sessionDuration(now.plusDays(5), now.plusDays(30))
+                .enrollment(Enrollment.createFreeEnrollment(new Students(List.of(NsUserTest.JAVAJIGI))))
+                .build();
```

Comment:
```suggestion
        Session session1 = new Session.Builder(1L)
                .sessionType(SessionType.FREE)
                .build();
```
빌더 패턴을 적용해 테스트하는 것이 Session 생성을  편하게 하기 위함이다.
무료 강의 테스트라면 해당 값을 식별할 수 있도록 위와 같이 구현하면 어떨까?